packages/components/src/components/Divider/Divider.tsx

```diff
@@ -23,6 +26,9 @@ export type DividerProps = AllowedFrameProps & {
     orientation?: DividerOrientation;
     strokeWidth?: number;
     color?: Color;
+    children?: ReactNode;
+    contentPosition?: 'start' | 'center' | 'end';
```

Comment:
Please, can you add comment "TODO unify with uiAlignments in the future"? I'm going to refactor these: 

```
export const uiHorizontalAlignments = ['left', 'center', 'right'] as const;
export type UIHorizontalAlignment = (typeof uiHorizontalAlignments)[number];

export const uiVerticalAlignments = ['top', 'center', 'bottom'] as const;
export type UIVerticalAlignment = (typeof uiVerticalAlignments)[number];

```

to this

```
export const uiAlignments = ['start', 'center', 'end'] as const;
export type UIAlignment = (typeof uiHorizontalAlignments)[number];
```

and it makes sense to use it here.
It's not ready yet, but it will be soon.

Reply:
just leaving here this link: https://github.com/trezor/trezor-suite/pull/17804